app/models/changelog_data.php

```diff
@@ -46,9 +47,119 @@
     '3.8.1' => '2015-06-24',
     '3.9'   => '2015-10-08',
     '3.10'  => '2015-11-13',
+    '4.0'   => '2016-01-04',
 ];
 
 $changelog = [
+    '4.0' => [
+        [
+            'message' => ['Propose search suggestions as your type in the search field'],
+            'section' => ['end_user'],
+            'type'    => ['new'],
+            'issues'  => [551],
+            'authors' => ['pascal', 'flod'],
+        ],
+        [
+            'message' => ['Create a <a href="/rss/">RSS</a> feed for release notes'],
```

Comment:
If you read it from Thunderbird, I think it won’t work (isn’t it the reason why we updated all the relative links in this file?). If it works then we can probably revert the change we made